addon/modifiers/uk-modal.js

```diff
@@ -0,0 +1,57 @@
+import { registerDestructor } from "@ember/destroyable";
+import Modifier from "ember-modifier";
+import UIkit from "uikit";
+
+export default class UkModalModifier extends Modifier {
+  #modal;
+  #events;
```

Comment:
What do the # do?

Reply:
It's a private class field: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields

Reply:
Oh cool, TIL